packages/blocks/src/image-block/image-model.ts

```diff
@@ -24,6 +40,33 @@ export const ImageBlockSchema = defineBlockSchema({
     role: 'content',
   },
   transformer: () => new ImageBlockTransformer(),
+  toModel: () => {
+    return new ImageBlockModel();
+  },
 });
 
-export type ImageBlockModel = SchemaToModel<typeof ImageBlockSchema>;
+@RectElement
```

Comment:
This decorator duplicates the concept of *element*. Is it indicating top-level content in edgeless mode? If so, it's also weird that shape, connector and frame are not decorated by it. Could you elaborate?

Reply:
It is Mixin, some utils of rect-like. so changing a name would be great.